### PR TITLE
Upgrade handlebars template

### PR DESCRIPTION
**Pull request by matteofigus**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,11 +48,11 @@
...
-    "oc-template-handlebars": "6.0.13",
+    "oc-template-handlebars": "^6.0.14",
     "oc-template-jade": "6.0.12",
     "stringformat": "0.0.5"
   },
   "engines": {
     "node": ">=6"
   }
-}
+}
```
Comment (mattiaerre) [inline package.json]: add newline, please 😊

File: package.json [hunk 2]
```diff
@@ -48,11 +48,11 @@
     "oc-client-browser": "1.3.4",
     "oc-empty-response-handler": "1.0.0",
     "oc-template-es6": "^1.0.1",
-    "oc-template-handlebars": "6.0.13",
+    "oc-template-handlebars": "^6.0.14",
```
Comment (mattiaerre) [inline package.json]: w/ `caret`?

File: package.json (hunk 1 above)
Comment (matteofigus) [reply package.json]: done

File: package.json (hunk 2 above)
Comment (matteofigus) [reply package.json]: done

**Pull request merged**